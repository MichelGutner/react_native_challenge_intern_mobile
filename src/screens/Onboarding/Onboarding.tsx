import React from 'react';
import { SafeAreaView, Text } from 'react-native';

function Onboarding() {
  return (
    <SafeAreaView>
      <Text>Ola</Text>
    </SafeAreaView>
  );
}

export default Onboarding;
