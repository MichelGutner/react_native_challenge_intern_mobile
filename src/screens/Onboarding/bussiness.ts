import { useNavigation } from '@react-navigation/native';
export const titleMessage =
  'Temos uma solução para cada etapa da sua carreira na medicina';
export const subTitleMessage =
  'Existimos para te ajudar em cada passo dessa incrível missão de cuidar de vidas.';
export const titleButton = 'Começar Agora';