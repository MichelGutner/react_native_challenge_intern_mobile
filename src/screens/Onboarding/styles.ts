import { TouchableOpacityProps } from 'react-native';
import styled from 'styled-components/native';
import { Themes } from '../../../themes/themes';

type TouchableType = TouchableOpacityProps;

export const Container = styled.SafeAreaView`
  flex: 1;
  background-color: ${Themes.colors.white};
`;

export const ImageLogoOnboarding = styled.View`
  left: 104px;
  top: 64px;
`;

export const TextTitle = styled.Text`
  width: 328px;
  height: 55px;
  left: 16px;
  top: 141px;

  font-family: Red Hat Display;
  font-style: normal;
  font-weight: bold;
  font-size: 20px;

  display: flex;
  align-items: center;
  text-align: center;
  color: ${Themes.colors.TitleOnboarding};
`;

export const TextSubTitle = styled.Text`
  width: 328px;
  height: 44px;
  left: 16px;
  top: 170px;

  font-family: Red Hat Display;
  font-style: normal;
  font-weight: normal;
  font-size: 14px;

  text-align: center;
`;

export const IllustrationImage = styled.View`
  top: 200px;
`;

export const OnboardingButton = styled.TouchableOpacity<TouchableType>`
  width: 312px;
  height: 48px;
  left: 24px;
  top: 282px;
  align-items: center;
  justify-content: center;

  background-color: ${Themes.colors.buttonOnboarding}

  border-radius: 4px;
`;

export const TitleButton = styled.Text`
  font-family: Roboto;
  font-style: normal;
  font-weight: 500;
  font-size: 16px;
  color: ${Themes.colors.white};
`;
