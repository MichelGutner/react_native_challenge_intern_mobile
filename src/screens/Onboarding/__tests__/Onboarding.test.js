import React from 'react';
import renderer from 'react-test-renderer';
import Onboarding from '../Onboarding';

test('renders correctly', () => {
  const tree = renderer.create(<Onboarding />).toJSON();
  expect(tree).toMatchSnapshot();
});