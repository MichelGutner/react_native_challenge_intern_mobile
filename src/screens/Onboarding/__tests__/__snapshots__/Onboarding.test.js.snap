// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "blue",
      "flex": 1,
    }
  }
>
  <Text>
    Ola
  </Text>
</RCTSafeAreaView>
`;
